gramps/gen/filters/rules/place/_inlatlonneighborhood.py

```diff
@@ -145,6 +145,8 @@ def apply(self, db, place):
 
         # now we know at least one is given in the filter and is valid
 
+        place = self.get_object(data)
```

Comment:
Can you use data["lat"] and data["long"] to avoid constructing the object?

Reply:
Done!